varinfo/cmr_search.py

```diff
@@ -190,3 +190,21 @@ def get_edl_token_header(auth_header: str, cmr_env: CmrEnvType) -> str:
     else:
         edl_auth_header = auth_header
     return edl_auth_header
+
+
+def get_dmr_xml_url(granule_response: Sequence) -> str:
+    """Gets the OPeNDAP url from a CMR granule response and appends `.dmr.xml`"""
+    granule_link = next(
+        (
+            link['href']
+            for link in granule_response[0].get('links', [])
+            if link['rel'].endswith('/service#')
+            and 'inherited' not in link  # and 'opendap' in link
```

Comment:
Should that last condition be commented out? (It feels like it should either be enabled or removed)

If it is removed, how do we know that it's only an OPeNDAP service link that is being retrieved, what about other service links? What are the things that make an OPeNDAP URL unique in the `.json` response from CMR? (I could tell you for the `.umm_json` response, but that's not what we're dealing with here)